posthog/hogql_queries/experiment_funnel_query_runner.py

```diff
@@ -24,17 +28,17 @@ def calculate(self) -> ExperimentFunnelQueryResponse:
         return ExperimentFunnelQueryResponse(insight="FUNNELS", results=results)
 
     def _process_results(self, funnels_results: list[list[dict[str, Any]]]) -> dict[str, ExperimentVariantFunnelResult]:
-        variants = self.query.variants
+        variants = self.feature_flag.variants
```

Comment:
forgive me if I'm out of the loop, but what's the ramifications/rationale behind moving the variants to come from the feature flag instead of the query?

Reply:
The way I think about this - the experiment (and its linked flag) is the source of truth for experiment variants. We need to retrieve it in the query runner for other things too, so might as well retrieve the variants from there, no need to pass via the query.